<launch>
    
    <node name="robot_fts_transform" pkg="tf2_ros" type="static_transform_publisher" args="0.00 0.00 0.00 0.00 0.00 0.895849974346 -0.444356639946 fts_base_link fts_reference_link" />
    
    <node name="robot_camera_body_transform" pkg="tf2_ros" type="static_transform_publisher" args="0.15 0 1.05 3.14 -0.3 0.0 base_link camera_body_link" />
    <node name="camera_body_optical_transform" pkg="tf2_ros" type="static_transform_publisher" args="0 -0.02 0 0 0 0 1 camera_body_link camera_body_depth_frame" />
    <node name="camera_body_depth_optical_frame_transform" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 -0.5 0.5 -0.5 0.5 camera_body_depth_frame camera_body_depth_optical_frame" />
    <node name="base_to_rgb_optical" pkg="tf2_ros" type="static_transform_publisher" args="0.150 0.045, 1.05 -0.420 -0.420 0.569 0.570 base_link camera_body_rgb_optical_frame " />

    <node pkg="gait_parameters_estimation" name="gait_estimation" type="gait_estimation_node.py" output="screen">
    	<rosparam command="load" file="$(find gait_parameters_estimation)/include/config.yaml"/>
    </node>

    <node pkg="gait_parameters_estimation" name="gait_com_estimation" type="gait_parameters_estimation_com_tracker_node" output="screen">
    	<rosparam command="load" file="$(find gait_parameters_estimation)/include/KalmanFilterParameters.yaml"/>
    </node>
    <!-- include file = "$(find camera_lower_leg_tracking)/launch/toe_detection.launch" /-->

</launch>
